k8s/plugin/src/resources/upgrade.rs

```diff
@@ -60,7 +64,10 @@ impl UpgradeResources {
                     Ok(sa) => {
                         if sa.iter().count() == 0 {
                             let ns = Some(ns.to_string());
-                            let service_account = objects::upgrade_operator_service_account(ns);
+                            let service_account = objects::upgrade_operator_service_account(
```

Comment:
This file is getting quite unwieldy, especially the install, consider splitting it up in modules perhaps (not for this PR).